docs/volto/bootstrap.md

```diff
@@ -24,63 +24,84 @@ Make sure you set a CORS policy or things tend to magically go wrong. See <https
 
 (install-deps-volto-label)=
 
-## Installing Dependencies
+## Installing Prerequisites
 
-First step is to install the correct Node version using `nvm`:
+First step is to install the correct Node version using `nvm`, as indicated [here](https://6.dev-docs.plone.org/volto/getting-started/install.html#install-nvm-nodejs-version-manager).
```

Comment:
An excerpt from [“Here” Conceals What Users are Clicking](https://uxmovement.com/content/why-your-links-should-never-say-click-here/).

> The problem with using “here” in a link is that it conceals what the user is clicking.

```suggestion
First [install the correct Node version using `nvm`](https://6.dev-docs.plone.org/volto/getting-started/install.html#install-nvm-nodejs-version-manager).
```

Reply:
Done.